elastic/logstash
# fix pqrepair log4j setting

Fixed: #13725



## Release notes


Fix `pqrepair` unable to output log message

## What does this PR do?



This PR turns on `info` log level and add log4j properties to test classpath to enable log.
Also, add log message to let users know the status of repair.


## Why is it important/What is the impact to the user?



`pqrepair` does not print any log during the process, which is hard to trace PQ issue. 

## Checklist




## Author's Checklist



## How to test this PR locally



you should see log in console by running the following command
$ bin/pqrepair /any/path/to/pq/dir


## Related issues


- 

## Use cases



## Screenshots



## Logs
